.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "sw/ext/opencv_bebop"]
 	path = sw/ext/opencv_bebop
 	url = https://github.com/tudelft/opencv_bebop.git
+[submodule "sw/ext/librtcm3"]
```

Comment:
This is a fundamental question: do you want to allow re-use of code in other projects? Than separate libraries are the way to go. But submodules are probably not the way to go.

What about if I make a very simply package manager kind of app that can be used to copy-paste folders from a remote github account with a specific version, so we do store where the remote project is located and which version is the latest tested, but we do not need to import the entire project. For more complex subprojects like chibi-OS we can still use a submodule.

-e.g. librtcm: lib-only and not the documentation
-e.g. iridium-dialer: python script only and not the hardware